package.json

```diff
@@ -104,6 +104,7 @@
     "@apollo/client": "3.3.12",
     "@callstack/react-theme-provider": "^2.1.0",
     "@expo/vector-icons": "^12.0.4",
+    "@react-native-async-storage/async-storage": "^1.14.1",
```

Comment:
why are there two async-storage?

Reply:
@react-native-community/async-storage not found at npmjs.com 
I deleted it.